scripts/update_review_app.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -ev
+
+# Bail out of script on first expression failure and echo the commands as
+# they are being run.
+set -ev
+
+NAME="$1"
+
+if test -z $NAME; then
+  echo "You didn't provide a shell argument, so NAME isn't meaningful, exiting."
+  exit 1
+fi
+
+hokusai registry push --force --skip-latest --overwrite --verbose --tag $NAME
+hokusai review_app setup $NAME
```

Comment:
Is the `setup` step necessary for an existing review app?

Reply:
Weirdly, it is. I tried to elaborate in the commit message for https://github.com/artsy/force/pull/5370/commits/116e3e01ca810fc3d0cb0a14be40ec3e8dfb7b4e 

```
* Create a review app YAML file before deploying it.
  + Because review app management in rooted in a file on the FS, we need to
    have a "stub" review app YAML file to effectively re-deploy code to a
    review app.
```